core/web/job_runs_controller.go

```diff
@@ -69,6 +69,10 @@ func (jrc *JobRunsController) Create(c *gin.Context) {
 	}
 }
 
+func webInitiatedMeta() *models.JSON {
+	return &models.JSON{}
+}
```

Comment:
maybe remove?

Reply:
The alternative would be to use `&models.JSON{}` inline on 63. Do you think that would be clearer?

Reply:
For sure, you will be running into this pattern quite often in the code base.